src/packages/core/src/external/githubCorner.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+export interface GithubCornerProps {
+  githubUrl: string;
+  fill?: string;
+  color?: string;
+  width?: number;
+  height?: number;
+}
+
+interface StyledSvgProps {
```

Comment:
we currently prefix all interfaces with `I` so this should be `IStyledSvgProps`

Reply:
Done